pkg/api/query_api_ops.go

```diff
@@ -14,6 +14,19 @@ var queryTags = []string{"Query"}
 
 // RegisterQueryAPI registers all query related endpoints
 func RegisterQueryAPI(a huma.API, caller string, querier query.Runner, middlewares huma.Middlewares) {
+	// query running
+	huma.Register(a,
```

Comment:
This section exists below for the distributed query, but seems to have been forgotten here...